ipyvizzu.py

```diff
@@ -126,15 +126,36 @@ def dump(self):
 
 class Chart:
     """
-    Wrapper over vizzu Chart
+    Wrapper over Vizzu Chart
     """
 
-    def __init__(
-        self, vizzu_url="https://cdn.jsdelivr.net/npm/vizzu@latest/dist/vizzu.min.js"
-    ):
-        self._vizzu_url = vizzu_url
+    def __init__(self):
+        self._vizzu = "https://cdn.jsdelivr.net/npm/vizzu@latest/dist/vizzu.min.js"
+        self._div_style = "width:800px; height:480px;"
         self._calls = []
 
+    @property
+    def vizzu(self):
```

Comment:
Do we need to read this value?

Reply:
No, at the moment, but is it a problem, if we can?

Reply:
I think `vizzu` as constructor parameter shows better its nature, because it is evaluated only once and it has no effect to the method calls. I think property set/get would be beneficial if the change of the `vizzu` property would effect to the next `animate` call.

We can split the constructor parameters into separated properties, but it makes it harder to follow the effect of the default values.

Reply:
@nyirog And what do you think about this:
- the url of vizzu, the width and height of div belongs to Chart.show()
- these params do not affect any other part of Chart like Chart.init, Chart.animate
- so these could be params of chart.show and
- this could be a real use case, create a Chart, with anims, and play it with different sizes or Vizzu versions

chart = Chart()
chart.animate(...)
...
chart.animate(...)

chart.show(width="800px", height="400px")

chart.show(width="400px", height="800px")

Reply:
With the method name `show` I tried to mimic the API of [matplotlib.Figure](https://matplotlib.org/stable/api/figure_api.html#matplotlib.figure.Figure.show) where `show` does not accept any configuration only opens the figure.

However the other source of ideas is [plotly](https://plotly.com/python/renderers/) where the `show` method accepts `width` and `height` parameter. So it is not uncommon to configure the renderer with the `show` method. We can follow plotly way, too.

Reply:
I place them in the constructor cause we discuss some new feature with @simzer and based on this they fit better there.